src/Console/Command.php

```diff
@@ -63,6 +63,10 @@ public function __construct()
         $this->modelFactory('Table', function ($alias) {
             return $this->getTableLocator()->get($alias);
         });
+
+        if (isset($this->modelClass)) {
```

Comment:
Loading the model in constructor itself isn't lazy loading. Shouldn't the __set() magic method be overridden for that like it's done in controller for that?

Reply:
For shells and commands nano opt lazyloading sounds wrong.

Reply:
I don't think we need lazy loading either. Commands don't really need the same performance profile that web controllers do.